tests/testing_common.py

```diff
@@ -905,7 +905,7 @@ def _test_delete_adapter(self, model_id, config_cls, config_kwargs):
         self.assertFalse(adapter_to_delete in model.peft_config)
         self.assertEqual(model.active_adapters, ["default"])
 
-        key_list = [key for key, _ in model.named_modules() if "lora" not in key]
+        key_list = [key for key, _ in model.named_modules() if model.prefix not in key]
```

Comment:
Good amendment, but this made me curious why the code worked at all, because `"lora"` was hard-coded but we also test LoHa and LoKr. It turns out the that prefix check is totally unnecessary and the code can be simplified to:

`key_list = [key for key, _ in model.named_modules()]`

Could you please fix that? Same for `_test_delete_inactive_adapter` and `_test_weighted_combination_of_adapters`. Thanks.

Reply:
I was also curious about why this worked at all. I'll fix it!